src/plugins/data/public/query/query_string/language_service/query_result.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import './_recent_query.scss';
+
+import { BehaviorSubject } from 'rxjs';
+import { EuiButtonEmpty, EuiPopover, EuiText, EuiPopoverTitle } from '@elastic/eui';
+
+import React, { useState, useEffect } from 'react';
+
+export interface QueryStatus {
```

Comment:
can you add this to the languages_service/types.ts

Reply:
added